### ISSUE TITLE
More keyboard binds

### ISSUE THREAD

**Issue by nunocastromartins** [jeremyramin/terminal-plus#64]
First of all, thank you for your work on this extension!
It is so far the most useful terminal extension I've found for Atom.

I'll open a series of issues that I have found while using terminal-plus.
I hope they're useful and help make this extension better!

The first is a request to add the following keyboard commands:
- Close all terminals
- Rename active terminal

**Comment by nunocastromartins** [comment]
Another useful one (mentioned in another issue, but I thought I'd include it here):
- Toggle between tab/bottom-panel mode

And maybe (although I believe there shouldn't even exist a way to fullscreen terminals, as I proposed at #66):
- Toggle fullscreen

**Comment by jeremyramin** [comment]
@nunocastromartins I can add those commands, but I probably will not bind them to anything by default. As for the fullscreen, I'll comment on that issue in the that thread.

**Comment by nunocastromartins** [comment]
Yes, that is what I meant! Please do add the commands, no need for them to have default bindings! :smile:

**Comment by jeremyramin** [comment]
I added `terminal-plus:close-all` and `terminal-plus:rename` in v0.12.4.
